src/views.rs

```diff
@@ -149,9 +149,17 @@ pub struct EncodableApiTokenWithToken {
     pub last_used_at: Option<NaiveDateTime>,
 }
 
+#[derive(Deserialize, Serialize, Debug)]
+pub struct OwnedCrate {
```

Comment:
I just realized I said "tuple" in the implementation instructions and you correctly inferred that this info should be in a struct instead like everything else 😅 Great work figuring out what I meant instead of what I said!